app.js

```diff
@@ -196,4 +196,25 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     word.textContent = 'The email must be in lowercase';
   }
-});
\ No newline at end of file
+});
+
+const inputName = document.getElementById('name');
+const inputText = document.getElementById('mail');
```

Comment:
- Please correct this assignment of values. The `inputText` is receiving the `mail` data. Your portfolio is your visit card and must present your best work without error.

Reply:
everything now is perfect